students/Hebdowska_Grazyna/lesson_06/frequency_analysis.py

```diff
@@ -0,0 +1,39 @@
+from operator import itemgetter
+
+
+def insert_text(n):
+    new_text = []
+    for i in range(n):
+        new_ = input()
+        new_ = new_.split(' ')
+        new_text.append(new_)
+    return new_text
+
+
+def frequency(some_text):
+    frequwncy_ = {}
```

Comment:
PEP8 warning
There is no need to change variable name to be different than function name
frequwncy_  -> frequency